src/cladetime/util/reference.py

```diff
@@ -60,6 +61,66 @@ def _docker_installed():
     return docker_enabled
 
 
+def _get_metadata_from_hub(date: datetime) -> dict:
+    """
+    Retrieve ncov metadata from variant-nowcast-hub archives when
+    Nextstrain S3 does not have historical metadata_version.json files.
+
+    The variant-nowcast-hub maintains weekly archives of modeled clades
+    with embedded ncov pipeline metadata dating back to September 2024.
+
+    Parameters
+    ----------
+    date : datetime
+        The date to retrieve metadata for (UTC)
+
+    Returns
+    -------
+    dict
+        ncov metadata dictionary with keys:
+        - nextclade_dataset_name_full
+        - nextclade_dataset_version
+        - nextclade_version
+        - nextclade_version_num
+
+    Raises
+    ------
+    ValueError
+        If no archive is found within 30 days before the requested date
+    """
+    date_str = date.strftime("%Y-%m-%d")
```

Comment:
if `date_str` is before 2024-10-09, then it won't find the metadata it is looking for. throw an error immediately.